### DISCUSSION TITLE
Academic domain verification

### DISCUSSION THREAD

**Discussion by gustko** [RosettaCommons/rosetta#244] [ROSIE] [answered]
Hi,
I can't get access to ROSIE as my email address is not recognized as academic. I'm working at the Norwegian University of Life Sciences for which the email domain is @nmbu.no. Would it be possible to add this domain manually? Would be greatly appreciated.

cheers,
Gustav

**Comment by lyskov** [comment, marked as answer]
— Thank you for letting us know! This should be fixed now, if you still have issues logging-in please feel free to re-open this issue.
